### ISSUE TITLE
aws_ec2_client_vpn_authorization_rule timeout after 1m0s

### ISSUE THREAD

**Issue by connor-tyndall** [terraform-providers/terraform-provider-aws#14156]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

v0.12.24

### Affected Resource(s)

* aws_ec2_client_vpn_authorization_rule

### Terraform Configuration Files

```hcl
resource "aws_ec2_client_vpn_authorization_rule" "custom" {
  for_each               = var.use_default_authZ_rules ? {} : var.authorization_rules
  client_vpn_endpoint_id = aws_cloudformation_stack.client_vpn.outputs["clientVpnId"]
  target_network_cidr    = each.value.cidr
  access_group_id        = each.value.ad_group_id
}
```

### Expected Behavior

The Terraform resource should include a `timeouts` argument or wait until the authorization rule moves to an "Active" state. 

### Actual Behavior

Error: error waiting for Client VPN authorization rule to be active: timeout while waiting for state to become 'active' (last state: 'authorizing', timeout: 1m0s)

### Steps to Reproduce

1. `terraform apply`

### References

* https://github.com/terraform-providers/terraform-provider-aws/pull/13950

**Comment by connor-tyndall** [comment]
@gdavison

**Comment by kumar-shantanu** [comment]
+1

**Comment by connor-tyndall** [comment]
Also, when modifying an existing authorization rule, the resource times out as well... which requires a manual deletion and reapply. 

`Error: error revoking Client VPN authorization rule: timeout while waiting for resource to be gone (last state: 'revoking', timeout: 1m0s)`

**Comment by nijave** [comment]
Yup, seems like the timeouts are set too aggressively https://github.com/terraform-providers/terraform-provider-aws/pull/13950/files#diff-b2b7576ebf9812ff7aa5592444112309R76

**Comment by connor-tyndall** [comment]
@gdavison @bflad  Any update on this? About 20% of authorization rules we create hit the 1 minute timeout. Can this be updated ASAP?

**Comment by ChristophShyper** [comment]
That would be very helpful. Cannot use this resource in automation, it fails always, just by few seconds.

**Comment by bflad** [comment]
The fix for this has been merged and will release with version 3.6.0 of the Terraform AWS Provider, later this week. Thanks to @connor-tyndall for the implementation. 👍
